### ISSUE TITLE
site 3037

### ISSUE THREAD

**Issue by roblanf** [W-L/ProblematicSites_SARS-CoV2#2]
Hi there,

In our BAM files we noticed (independently of your analyses) that site 3037 looks a lot like an RNA degredation site. We see batch effects on samples we've processed, and samples with a lot of heterozygosity for T/C.

I note that in your original post you state that 3037 should be masked, and in the phylogenetic information plot it has a delta of ~0 (and you've coloured it red).

However, site 3037 doesn't appear in any of your mask files as far as I can tell (unless, as before, I've missed something simple).

Can you clarify what's happening here? 

Many thanks,

Rob

P.S. I'm now using your mask file to estimate regularly-updated trees, so if it's OK with you I am hoping to treat your mask file as a kind of public list of best-practice sites to mask. I'd prefer to do this than try to maintain a separate list, though it might mean I occasionally attempt to discuss individual sites here!)

**Comment by NicolaDM** [comment]
Hi Rob, thanks for the heads up! Indeed it's very valuable for us to receive feedback like this!
We do wish to provide this list as a resource for others, but of course our analyses are far from perfect, so we do appreciate contributions like yours. 

In the old analysis we were masking some sites that had very high prevalence in addition to being homoplasic - you can find the old files here: 
https://github.com/W-L/ProblematicSites_SARS-CoV2/tree/master/archived_vcf

However, in the new version, we tried not to mask variants that have such high prevalence and are likely to be mostly caused by true mutation events. It's not an easy choice, because there is a trade-off between removing some of the phylogenetic signal from the true mutation events, and removing the noise from the artefacts, degradation, or just hypermutability, and the ideal masking might depend on the specific task at hand (e.g. phylogenetic inference vs inference of selection).

3037 seems indeed the most homoplasic site among the ones we do not filter, and as such it is one of the most suspicious ones. There are also some ambiguity characters at this position (23 "y" and 4 isolated "n").
However, this mutation has also extremely high prevalence (10820 "t" and 5028 "c"), and as such, if this was a normal high frequency mutation, some level of ambiguity and homoplasy would be expected just because of contamination, recombination, mixed infections, etc.
What seems plausible, is that this position is hypermutable, due in particular to its context ("TTCT" -> "TTTT") and to APOBEC. This mutation seems to appear indiscriminately in many different labs, both as fixed substitution as well as ambiguity character, so this is unlikely to be specifically any sequencing artefact.

Therefore, so far we suspect this site to be more "weird" (in particular hypermutable) than "wrong" (in particular a sequencing artefact), but, as always, both things might be going on at the same time, so we are happy to receive evidence that might tip the balance in one direction.
Given your warning and the pre-existing evidence, we will now mark the site as "caution", and we can further move it to "mask" with more evidence.

As a visual reference, this is the substitution pattern inferred by NextStrain:
https://nextstrain.org/ncov/global?c=gt-nuc_3037

**Comment by roblanf** [comment]
Our samples come from all over the tree, so batch effects at a single site are not expected. Still, I wouldn't count evidence like this without some more details, so perhaps what we should do from our end is work on properly documenting and testing the batch effects. And as you point out (nice link, btw!) that site does seem to match the tree very well. 

I think maybe what we should do is to look more carefully at the batch effects at our end. Though if it is batch effects, then it's much harder to explain that signal in the tree!

Putting in caution but not mask seems like a good idea for now.

**Comment by roblanf** [comment]
Hi guys,

Just a note - in a new analyses we've noticed that 3037 seems like an odd site. (Actually, someone else noticed it, I had forgotten I'd already posted here). I don't have any additional evidence for you, but I did notice that it still hasn't been added to the caution vcf. 

Could you update the vcf?

Rob

**Comment by NicolaDM** [comment]
Hi Rob,
We are now preparing a 3rd version of the file, including in particular results from Russ Corbett and team:
https://www.biorxiv.org/content/10.1101/2020.06.08.141127v1.abstract
We wanted to publish the new version at the same time as an update on virological explaining the changes and their rationale.
We are now ironing out the details, and we were hoping to post this probably next week, would it be ok?

Regarding 3037, what kind of explanation you think we should give? 
Should we denote you as the source of this analysis?

Best wishes,
Nicola

**Comment by roblanf** [comment]
Sounds great. 

I'm not sure I've really added much to the discussion on site 3037. I have to say I trust your analyses, and I think that between you guys and Russ Corbett's analyses, we have most things covered as well as possible. I don't have strong evidence to exclude site 3037, and most of the evidence that makes me worry about it is that it's marked as 'exclude' in your original analysis.

It does look like an RNA degredation target (or maybe in APOBEC site??), insofar as many of our BAMs have believable mixtures of genotypes at that site. But if that were the only cause of differences I wouldn't expect the Nextstrain trees to partition the site so cleanly. 

So, to sum up, I'm still a little confused on this site. Seems sensible to mark it as 'caution', but not to exclude for now. 

Feel free to mention me as the source of some of the analysis or not, as you see fit. I'm unable to provide solid evidence (even BAMs) since for now the data are covered by privacy rules beyond my control, and I don't have the time to go into published sequencing data and see if the same patterns exist there. My evidence is, therefore, at best circumstantial!

**Comment by NickGoldman** [comment]
We may mark as "caution". The update is waiting on Nicola getting back to work after a few days' leave and verifying some stuff, possibly with the UCSC folk.  It would be good to have some virological.org comments explaining some of the reasoning behind the updates too.
